pkg/configure/oneagent/pmc/ruxit/types.go

```diff
@@ -35,12 +58,53 @@ func (pmc ProcConf) ToMap() ProcMap {
 	return sections
 }
 
-// ToString creates the content of the configuration file, the sections and properties are printed in a sorted order, so it can be tested.
-func (pmc ProcConf) ToString() string {
-	rawMap := pmc.ToMap()
+// ProcMap presents the content in a more easy to work with format. (a map of maps).
+type ProcMap map[string]map[string]string
 
+var (
+	redundantEntries = map[string][]string{
+		"general": {"logDir", "dataStorageDir"},
+	}
+	additionalEntries = ProcMap{
+		"general": map[string]string{
+			"storage": "\"/var/lib/dynatrace/oneagent\"", // TODO: Make configurable?
+		},
+	}
+)
+
+func (pm ProcMap) SetupReadonly(installPath string) ProcMap {
```

Comment:
here you are doing good :)

Reply:
`s/installPath/installDir/g`

Reply:
`--install-path` is where it is coming from